extension.json

```diff
@@ -1,59 +1,64 @@
 {
 	"name": "Page Approvals",
-
 	"version": "1.0.0",
-
 	"author": [
 		"[https://Professional.Wiki/ Professional.Wiki]"
 	],
-
 	"license-name": "BSD-3-Clause",
-
 	"url": "https://github.com/ProfessionalWiki/PageApprovals",
-
 	"descriptionmsg": "pageapprovals-description",
-
 	"requires": {
 		"MediaWiki": ">= 1.39.0"
 	},
-
 	"MessagesDirs": {
 		"PageApprovals": [
 			"i18n"
 		]
 	},
-
 	"AutoloadNamespaces": {
 		"ProfessionalWiki\\PageApprovals\\": "src/",
+		"ProfessionalWiki\\PageApprovals\\HookHandler\\": "src/HookHandler/",
 		"ProfessionalWiki\\PageApprovals\\Tests\\": "tests/"
 	},
-
 	"Hooks": {
+		"OutputPageBeforeHTML": [
+			"ProfessionalWiki\\PageApprovals\\HookHandler\\PageApprovalsHooks::onOutputPageBeforeHTML"
+		]
 	},
-
-	"config": {
-	},
-
 	"ResourceFileModulePaths": {
-		"localBasePath": "resources",
-		"remoteExtPath": "PageApprovals/resources"
+		"localBasePath": "modules",
+		"remoteExtPath": "PageApprovals/modules"
```

Comment:
These 2 can stay `resources`. That's the newer naming convention. We have `modules` in ProAdmi because that extension was created quite some time ago, but otherwise we've been using `resources` in all newer extensions.